relay-server/src/metrics_extraction/snapshots/relay_server__metrics_extraction__event__tests__extract_span_metrics.snap

```diff
@@ -5882,6 +5884,7 @@ expression: metrics
         ),
         tags: {
             "environment": "fake_environment",
+            "messaging.destination.name": "default",
```

Comment:
Are the `default` values placeholders? Can we add a test with an expected payload to verify the tag value is correct?

Reply:
Hey @iker-barriocanal , these values are not placeholders, they come from the tests here:
https://github.com/getsentry/relay/blob/master/relay-server/src/metrics_extraction/event.rs#L1060
Thanks!